### ISSUE TITLE
Unpin versions?

### ISSUE THREAD

**Issue by moshemalawach** [21dotco/two1-python#29]
Hello,

Is it possible to unpin the == versions in setup.py?
It makes it very hard to use in conjunction with other libraries! (and two1 is required itself by a bunch of other libs...)

Thanks

**Comment by cheeseandcereal** [comment]
I agree, especially with stuff like `requests<=2.11.0`, and pinning exact alpha versions of packages (`protobuf==3.0.0a3`), it's nearly impossible to use this package with anything else without getting dependency conflicts and potential unknown behavior.

Not to mention that dev dependencies should be removed from the requires entirely (flake8, pytest, etc)
